index.js

```diff
@@ -43,11 +52,13 @@ module.exports = (function () {
 		return getHarness().only.apply(this, arguments);
 	};
 
+	/** @type {import('.').CreateStream} */
 	lazyLoad.createStream = function (opts) {
 		var options = opts || {};
 		if (!harness) {
 			var output = through();
-			getHarness({ stream: output, objectMode: options.objectMode });
+			// eslint-disable-next-line no-extra-parens
```

Comment:
Question: is this a bug or missing feature in `no-extra-parens`? If the parenthesis are meaningful, I'd expect it to either support this use case out of the box (bug?) or have an option to allow it (missing feature?).

Reply:
it's neither; the TS eslint no-extra-parens rule covers this case, but the core rule doesn't (and shouldn't) know about type comments or TypeScript's weird requirement here for the parens to do casting.

Reply:
Hmm, I wonder if this means you'd want to use the typescript-eslint rule... since this code is kind of in the TypeScript camp now. It happens not to use any TS-provided syntax, but is using JS syntax in a TS-specific way...

Reply:
yes, i probably should :-)

unfortunately eslint 8.9 broke my parsing use case, so i can't update my personal config to use flat config, so i can't transparently include other plugins, and i don't want to add TS eslint as a peer dep and force all of my projects to add it :-/